commands/system/vpn/vpn-status.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title VPN Status
+# @raycast.mode inline
+# @raycast.refreshTime 1m
+
+# Optional parameters:
+# @raycast.icon 📡
+
+# @Documentation:
+# @raycast.packageName VPN
+# @raycast.description Stop VPN connection.
+# @raycast.author Alexandru Turcanu
+# @raycast.authorURL https://github.com/Pondorasti
+
+
+source vpn-config.sh
+VPN=$VPN_NAME
+
+status=$(scutil --nc status "$VPN" | sed -n 1p)
+
+echo "$status"
```

Comment:
It would be nice to include the `$VPN` value in this output, so the dashboard command shows something like "Connected to `$VPN`".
```suggestion
if [ "$status" == "Connected" ]; then
  echo "$status to $VPN"
  exit 0
fi

echo "$status"
```

Reply:
Interesting idea. From my usage, I'm always going to be connecting to the same VPN, which makes the echo message too verbose and repetitive over time. 

At the end of the day, it comes down to style and aesthetics. Works either way for me.